index.html

```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
         </a>
       </div>
     </header>
+    <section class="works">
+      <div class="projects">
+        <img src="images/snapshoot-project-1.png" alt="snapshoot-project-1" />
+        <p class="project-title">Tonic</p>
```

Comment:
- [ ] Please know that each project title should be wrapped in a `hx` tag, kindly change that. 😄

Reply:
**Made the title with h2 tag** 👍. Hope that works